modules/extensions-support/src/main/java/org/mule/runtime/module/extension/internal/introspection/describer/AnnotationsBasedDescriber.java

```diff
@@ -192,24 +197,34 @@ private String getVersion(Extension extension)
 
     private void declareSubTypesMapping(ExtensionDeclarer declaration, Class<?> extensionType)
     {
-        SubTypesMapping typesMapping = extensionType.getAnnotation(SubTypesMapping.class);
+        List<SubTypeMapping> typeMappings = parseRepeatableAnnotation(extensionType, SubTypeMapping.class, c -> ((SubTypesMapping)c).value());
```

Comment:
Can `c -> ((SubTypesMapping)c).value()` be replaced by `SubTypesMapping::value` ?